package.json

```diff
@@ -115,7 +115,7 @@
     "test:integration:debug": "TZ='UTC' run-p --race svelte:watch proxy:start:test-watch wait:debug",
     "test:unit": "jest",
     "test:unit:watch": "jest --watchAll",
-    "wait:test": "wait-on tcp:8080 && yarn svelte:build && yarn cypress:run && kill `pidof api`",
+    "wait:test": "wait-on tcp:8080 && yarn svelte:build && yarn cypress:run; status=$?; kill `pidof api`; exit $status",
```

Comment:
Is there a way to run this workaround only on the Linux CI machine? I had to install `pidof` via brew on mac-n-cheese and my local machine for this to work. This might break running tests for other mac users like @MeBrei.

Reply:
Let’s see if this works: https://github.com/radicle-dev/radicle-upstream/pull/1185/commits/d9d2f0654e2a869cfe958d199ffbdfdafd5b3fb6